Libraries/PushNotificationIOS/RCTPushNotificationManager.mm

```diff
@@ -121,7 +121,7 @@ @interface RCTPushNotificationManager () <NativePushNotificationManagerIOS>
 
 @implementation RCTPushNotificationManager
 
-#if !TARGET_OS_UIKITFORMAC && !TARGET_OS_OSX
+#if !TARGET_OS_UIKITFORMAC && !TARGET_OS_OSX // [macOS]
```

Comment:
Just a thought but I wonder for if defs like this if it would make more sense for the !TARGET_OS_OSX to be moved to the next line. Like 

#if !TARGET_OS_UIKITFORMAC
#if !TARGET_OS_OSX // [macOS]
...

or vice versa. It's not clear to me if the TARGET_OS_UIKITFORMAC part was always there or just for macOS (although, just realized maybe it's only for macOS, in which case do we need another // [macOS] tag at the endif of this block

Reply:
`TARGET_OS_UIKITFORMAC` is from RN Core. It's a check to compile for Catalyst, where you make an iOS app run on macOS. RN Core does support Catalyst, so there's a few of these around. I'm gonna prefer to keep the logic as is, in case the change does affect something.